src/redux/books/books.js

```diff
@@ -1,25 +1,58 @@
+/* eslint-disable no-restricted-syntax */
```

Comment:
- [ ] Let's not disable linter errors. These are here to help us write best code.

Reply:
The object of arrays in objects is so complex to access, would you please help to point me to a resource where I can find another way to do it rather than using a for loop